variables.tf

```diff
@@ -79,7 +79,7 @@ variable "fips_enabled" {
 variable "ssh_public_key" {
   description = "A custom ssh key to control access to the AKS cluster. Changing this forces a new resource to be created."
   type        = string
-  default     = "~/.ssh/id_rsa.pub"
+  default     = null
```

Comment:
Need new appropriate default value or local that would mirror what we had in the past.

Reply:
The changes for variable `ssh_public_key` were backed out. TFLint does not seem to be throwing the error previously received. Pushing up for re-review.